lms/templates/discussion/_discussion_module.html

```diff
@@ -1,11 +1,14 @@
+<%page expression_filter="h"/>
 <%include file="_underscore_templates.html" />
 <%!
 from django.utils.translation import ugettext as _
 %>
 
-<div class="discussion-module" data-discussion-id="${discussion_id | h}" data-user-create-comment="${can_create_comment}" data-user-create-subcomment="${can_create_subcomment}" data-read-only="false">
-    <a class="discussion-show control-button" href="javascript:void(0)" data-discussion-id="${discussion_id | h}" role="button"><span class="show-hide-discussion-icon"></span><span class="button-text">${_("Show Discussion")}</span></a>
+<div class="discussion-module" data-discussion-id="${discussion_id}" data-user-create-comment="${can_create_comment}" data-user-create-subcomment="${can_create_subcomment}" data-read-only="false">
+    <a class="discussion-show control-button" href="javascript:void(0)" data-discussion-id="${discussion_id}" role="button">
+        <span class="show-hide-discussion-icon"></span><span class="button-text">${_("Show Discussion")}</span>
+    </a>
     % if can_create_thread:
-        <a href="#" class="new-post-btn" role="button"><span class="icon fa fa-edit new-post-icon"></span>${_("New Post")}</a>
+        <button class="new-post-btn btn-neutral btn-small">${_("Add a Post")}</button>
```

Comment:
Thanks for checking. The icon removal was intentional as the new design doesn't include an edit icon:

![image](https://cloud.githubusercontent.com/assets/5985072/15733808/b7bc7afe-2857-11e6-9583-5b13eb5120c7.png)